src/library/searchquery.cpp

```diff
@@ -107,6 +107,10 @@ NotNode::NotNode(QueryNode* pNode)
         : m_pNode(pNode) {
 }
 
+NotNode::~NotNode() {
+    delete m_pNode;
```

Comment:
:+1: for the usage of `std::unique_ptr`. 

Do you mind splitting your C++11 refactor into several PR's. We use raw pointers a lot in mixxx and several medium sized PR's are just easier to review.